internal/rpc/insecure.go

```diff
@@ -87,11 +86,8 @@ func (s *insecureServer) start(params *ServerParams) (func(), error) {
 		}
 	}
 
-	s.httpMux.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "ok")
-	})
+	s.httpMux.HandleFunc("/healthz", monitoring.NewHealthProbe(params.handlersForHealthCheck))
 	s.httpMux.Handle("/", s.proxyMux)
-	zpages.Handle(s.httpMux, "/debug")
```

Comment:
why are we removing zpage?

Reply:
`monitoring.bindZpages` already calls this.